airflow/config_templates/default_airflow.cfg

```diff
@@ -218,6 +218,14 @@ hide_paused_dags_by_default = False
 
 [email]
 email_backend = airflow.utils.email.send_email_smtp
+subject = Airflow alert: {{{{ti}}}}
```

Comment:
Sorry for the delay - I've reworked the PR to take `subject_template` and `html_content_template` config entries, each specifying a path to a template. Please let me know what you think.